ipi/inputs/barostats.py

```diff
@@ -42,10 +42,7 @@ class InputBaro(Input):
             {
                 "dtype": str,
                 "default": "dummy",
-                "help": """The type of barostat.  Currently, only a 'isotropic' barostat is implemented, that combines
-                                    ideas from the Bussi-Zykova-Parrinello barostat for classical MD with ideas from the
-                                    Martyna-Hughes-Tuckerman centroid barostat for PIMD; see Ceriotti, More, Manolopoulos, Comp. Phys. Comm. 2013 for
-                                    implementation details.""",
+                "help": """The type of barostat.""",
```

Comment:
Agreed, but I don't think I have the expertise to write that (yet). If one of the experts, maybe @venkatkapil24  drops a sentence each here, I'll be happy to incorporate it! Otherwise we can also wait until I've read up on it, but that could take some time :)

Reply:
If you like you can go for this:

As the name suggests, the 'isotropic' mode implements the Bussi-Zykova-Parrinello barostat [doi:10.1063/1.3073889] that isotrpically scales the volume while sampling the isothermal isobaric ensemble. The implementation details are given in [doi:10.1016/j.cpc.2013.10.027]. The 'sc-isotropic' mode implements the same for Suzuki-Chin path integral molecular dynamcis [10.1021/acs.jctc.8b01297]. This barostat is suitable for simulating liquids.  The flexible mode implements the path integral version of Martyna-Tuckerman-Tobais-Klien barotat which incorporates full cell fluctuations while sampling the isothermal isobaric ensemble  [doi:10.1063/1.478193]. This is suitable for anisotropic systems such as molecular solids. The 'anisotropic' mode implements the Raiteri-Gale-Bussi barostat which enables cell fluctuations at constant external stress [10.1088/0953-8984/23/33/334213]. It is suitable for simulating solids at given external (non diagonal) stresses and requires specifying a reference cell for estimating strain. Note that this ensemble is valid only within the elastic limit of small strains.  For diagonal stresses (or external pressures) the 'flexible' and the 'anisotropic' modes should give very similar results.

Reply:
Ok, the string from @venkatkapil24 is really good. Can you please add it so we merge this ASAP?

Reply:
Thanks @venkatkapil24 ! Added the information to both individual barostats and `InputBaro`.